superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -26,6 +26,19 @@ import {
 import { debounce } from 'lodash';
 import { matchSorter, rankings } from 'match-sorter';
 import Collapse from 'src/components/Collapse';
+import Alert from 'src/components/Alert';
+import { SaveDatasetModal } from 'src/SqlLab/components/SaveDatasetModal';
+import { exploreChart } from 'src/explore/exploreUtils';
+import moment from 'moment';
+import rison from 'rison';
+import {
+  DatasetRadioState,
+  EXPLORE_CHART_DEFAULT,
+  DatasetOwner,
+  DatasetOptionAutocomplete,
+  updateDataset,
+} from 'src/SqlLab/components/ResultSet';
```

Comment:
we should centralize this in a types file now instead of pulling it from `ResultSet`

Can you move this to `superset/superset-frontend/src/SqlLab/types.ts` and import from there

Reply:
Can do! Done in [`this commit`](https://github.com/apache/superset/pull/19855/commits/9985ec71158a6d7088f1382b24bfa1454d4891ea)